.github/workflows/kotlin.yml

```diff
@@ -188,6 +188,62 @@ jobs :
         with :
           report_paths : '**/build/test-results/test/TEST-*.xml'
 
+  jvm-stateChange-runtime-test :
```

Comment:
Woot! @RBusarow should own reviewing this for correctness, though.

Reply:
i did do some cleanup after this - https://github.com/square/workflow-kotlin/pull/1011